get-dependency-list.js

```diff
@@ -69,11 +70,12 @@ module.exports = function(filename, serverless, cache) {
   while (localFilesToProcess.length) {
     const currentLocalFile = localFilesToProcess.pop();
 
-    if (filePaths.has(currentLocalFile)) {
+    if (filePaths.has(currentLocalFile) || checkedFiles.has(currentLocalFile)) {
       continue;
     }
 
     filePaths.add(currentLocalFile);
+    checkedFiles.add(currentLocalFile);
```

Comment:
I suspect we don't really need two sets here, but I didn't want to mess with how `filePaths` is used.